server/lib/device/light/light.command.js

```diff
@@ -13,6 +13,7 @@ const { DEVICE_FEATURE_CATEGORIES, DEVICE_FEATURE_TYPES } = require('../../../ut
  * light.command(message, classification, context);
  */
 async function command(message, classification, context) {
+  let nbLightsAffected = 0;
```

Comment:
Doesn't it should be `nbAffectedLights`?

Reply:
Yes, fixed ✅